meta-balena-common/recipes-core/busybox/busybox_%.bbappend

```diff
@@ -3,5 +3,8 @@ SRC_URI_append = " \
     file://defconfig \
     file://balenaos.cfg \
     "
+SRC_URI_remove = " \
+    file://syslog.cfg \
+    "
 
```

Comment:
It's better to remove the dependency on busybox-syslog at distro level, for example with:
```
diff --git a/meta-balena-common/conf/distro/resin-systemd.conf b/meta-balena-common/conf/distro/resin-systemd.conf
index 5e548313563f..c27aa9f0554d 100644
--- a/meta-balena-common/conf/distro/resin-systemd.conf
+++ b/meta-balena-common/conf/distro/resin-systemd.conf
@@ -5,3 +5,4 @@ require conf/distro/include/balena-os.inc
 DISTRO_FEATURES_append = " systemd"
 VIRTUAL-RUNTIME_init_manager = "systemd"
 DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
+VIRTUAL-RUNTIME_base-utils-syslog = ""
```

Reply:
I don't intend to spend any more time on this. Ref https://github.com/balena-os/balena-dt-cloudconnector/pull/158#issuecomment-630358042

Reply:
hi @svenschwermer that's fine, I can take up the changes from here if you don't mind.

Reply:
Go ahead if it's functionally identical.